crates/config/src/network.rs

```diff
@@ -17,6 +17,8 @@ pub struct NetworkConfig {
     pub tx_gas_buffer: u16,
     #[serde(default = "default_tx_gas_price_buffer")]
     pub tx_gas_price_buffer: u16,
+    #[serde(with = "humantime_serde")]
+    pub reregistration_delay: Option<std::time::Duration>,
```

Comment:
I'd suggest we provide a default value if it's not there. I know we theoretically don't _need_ to re-register all the time, but the downside to doing it seems super minimal to stop any ghosts